cmd/admin-bucket-remote-add.go

```diff
@@ -239,10 +239,17 @@ func fetchRemoteTarget(cli *cli.Context) (sourceBucket string, bktTarget *madmin
 	if cli.IsSet("sync") && serviceType != string(madmin.ReplicationService) {
 		fatalIf(errInvalidArgument(), "Invalid usage. --sync flag applies only to replication service")
 	}
-	bandwidthStr := cli.String("bandwidth")
-	bandwidth, err := getBandwidthInBytes(bandwidthStr)
-	if err != nil {
-		fatalIf(errInvalidArgument().Trace(bandwidthStr), "Invalid bandwidth number")
+	var bandwidth uint64
+	if cli.IsSet("bandwidth") {
+		var err error
+		bandwidthStr := cli.String("bandwidth")
+		bandwidth, err = getBandwidthInBytes(bandwidthStr)
+		if err != nil {
+			fatalIf(errInvalidArgument().Trace(bandwidthStr), "Invalid bandwidth number")
+		}
+		if bandwidth < 100*1000*1000 {
```

Comment:
move this check inside getBandwidthInBytes(input, minThreshold) -> minThreshold is 100*1000*1000 if set to `-1` there is no limit.